### DISCUSSION TITLE
Public discussions for private repos

### DISCUSSION THREAD

**Discussion by andrewbanchich** [community/community#3736] [Discussions]
We have private repos for proprietary software, but I'd like to use Discussions for communicating with users who don't have access to the repo.

**Comment by sr229** [comment]
A workaround you can establish is create another repository for this case. Unfortunately Discussions is tied to the repository visibility,  as with every feature in GitHub relating to repositories.

**Reply by andrewbanchich** [reply]
Yep, which is why I'm giving some feedback saying I'd like that to not be the case. I think given that Discussions are meant for users (most devs don't need a Q&A for the software they wrote), it makes more sense to have them be open to the public in most cases.

**Reply by sr229** [reply]
This would require the repository to also be visible, which cancels the "private" visibility of the repository. I believe org-wide Discussions would serve a better purpose instead for this.

**Reply by andrewbanchich** [reply]
It *currently* requires it to be visible. I'm asking for that to change. This is a feature request.

**Comment by markcellus** [comment]
@andrewbanchich agreed. We have this same use-case. Just out of curiosity, do you think this same feature would make sense for GH Issues?

**Reply by andrewbanchich** [reply]
I think it makes sense for issues too, since users often need to file bug reports. However, issues make a lot of sense for internal stuff in private repos, but Discussions (to me) seem to have far less use if not public.
